library/src/lib/calendar/calendar.component.ts

```diff
@@ -689,29 +690,26 @@ export class CalendarComponent implements OnInit, OnDestroy, AfterViewChecked, C
         }
     }
 
-    validateDateFromDatePicker(date: Array<any>): boolean {
-        let isInvalid: boolean = false;
-        const month = date[0];
-        const day = date[1];
-        const year = date[2];
-        let numOfDaysInMonth = 0;
+    validateDateFromDatePicker(date: Date): boolean {
```

Comment:
Since we are returning a bool value, it's probably a good idea to give a proper name to this function.

Reply:
we're blowing it up anyways 🤷‍♀